packages/flutter/lib/src/material/toggle_buttons.dart

```diff
@@ -819,6 +819,7 @@ class _ToggleButton extends StatelessWidget {
         hoverColor: currentHoverColor,
         splashColor: currentSplashColor,
         focusNode: focusNode,
+        density: theme.density,
```

Comment:
Is this just to make sure that toggle buttons respond to changes in the theme's density values? In the future, would we need to to allow density to be passed in as a param to `ToggleButtons`?

Reply:
Yes, probably.  Removed this for now, so it won't respond to density.